### PR TITLE
Split modules

### PR DESCRIPTION
**Pull request by extsoft**
Implements #46.
More details in commit messages.

### CODE REVIEW

File: sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/SunshineTestNG.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
-package io.github.tatools.sunshine;
+package io.github.tatools.sunshine.testng;
 
+import io.github.tatools.sunshine.core.*;
```
Comment (Mras) [inline sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/SunshineTestNG.java]: Do we really need all those classes from `io.github.tatools.sunshine.core.*` here?

File: sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/LoadableTestNGSuite.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
-package io.github.tatools.sunshine;
+package io.github.tatools.sunshine.testng;
 
+import io.github.tatools.sunshine.core.*;
```
Comment (Mras) [inline sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/LoadableTestNGSuite.java]: Another `io.github.tatools.sunshine.core.*` import. Is this intended?

File: sunshine-junit4/src/main/java/io/github/tatools/sunshine/junit4/JunitSuite.java [hunk 3]
```diff
@@ -1,4 +1,6 @@
-package io.github.tatools.sunshine;
+package io.github.tatools.sunshine.junit4;
+
+import io.github.tatools.sunshine.core.*;
```
Comment (Mras) [inline sunshine-junit4/src/main/java/io/github/tatools/sunshine/junit4/JunitSuite.java]: Another 'star' import

File: sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/SunshineTestNG.java (hunk 1 above)
Comment (extsoft) [inline sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/SunshineTestNG.java]: Added #59

File: sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/LoadableTestNGSuite.java (hunk 2 above)
Comment (extsoft) [inline sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/LoadableTestNGSuite.java]: Added #59

File: sunshine-junit4/src/main/java/io/github/tatools/sunshine/junit4/JunitSuite.java (hunk 3 above)
Comment (extsoft) [inline sunshine-junit4/src/main/java/io/github/tatools/sunshine/junit4/JunitSuite.java]: Added #59

**Comment by extsoft** [comment]
@rultor please merge

**Pull request merged**